### PR TITLE
Dependencies update, code fixes, travis configuration fix

### PR DESCRIPTION
**Pull request by christianascone**
What I did:

- Updated Pod Opera spec (`Opera` to `OperaSwift`) with 2.0 version
- Changed every Opera reference (Opera to OperaSwift)
- Added missing URLRequestSetup protocol
- Fixed OTHER_SWIFT_FLAGS warning for Release config (Added `$(inherited)` flag)
- Updated RxSwift examples with new name convetion (for example `rx_object()` to `rx().object()`). I suggest you to take a look at this.
- Fixed travis configuration updating gems versions and Xcode version

### CODE REVIEW

**Review by samirthebti — APPROVED** [review]

File: Project-iOS/XLProjectName/XLProjectName/Networking/Manager.swift [hunk 1]
```diff
@@ -18,18 +18,18 @@ class NetworkManager: RxManager {
...
-        observers = [Logger()]
+        observers = [Logger() as OperaSwift.ObserverType]
     }
 
-    override func rx_response(_ requestConvertible: URLRequestConvertible) -> Observable<OperaResult> {
-        let response = super.rx_response(requestConvertible)
-        return SessionController.sharedInstance.refreshToken().flatMap { _ in response }
+    override func response(_ requestConvertible: URLRequestConvertible) -> PrimitiveSequence<SingleTrait, OperaResult> {
+        let response = super.response(requestConvertible)
+        return response
```
Comment (m-revetria) [inline Project-iOS/XLProjectName/XLProjectName/Networking/Manager.swift]: Please keep the call to the `refreshToken` function

**Review by m-revetria — COMMENTED** [review]
Hi @christianascone! Thanks for the contribution it looks really good. I left just a comment on. Please take a look and we will be glad to merge this PR.

**Comment by christianascone** [comment]
Thank you @m-revetria !
I just restored the call to the `refreshToken` function and checked Travis-CI status (build [passed](https://travis-ci.org/xmartlabs/Swift-Project-Template/builds/274804528?utm_source=github_status&utm_medium=notification)).

**Review by m-revetria — APPROVED** [review]
@christianascone thanks for fixing the comment!

**Pull request merged**